ludwig/constants.py

```diff
@@ -128,6 +128,7 @@
 NAME = "name"
 COLUMN = "column"
 TYPE = "type"
+OUTPUT_FLAG = "output_flag"
```

Comment:
I agree that it's better to get the output class straight from the config instead of looping through the features, I guess my initial thought was to change as little pre-existing code as possible so I tried to just add the flag so I could pick it up from the flattened feature list. If it's ok to pipe the config into all the preprocessors so I can access it at the `_preprocess_df_for_training` and `_preprocess_file_for_training` functions then I'll go ahead and make that change!